Python/ceval_macros.h

```diff
@@ -297,6 +333,12 @@ GETITEM(PyObject *v, Py_ssize_t i) {
 #define RECORD_BRANCH_TAKEN(bitset, flag)
 #endif
 
+#if _Py_TIER2
+#   define RECORD_DYNAMIC_JUMP_TAKEN() tstate->interp->jit_state.dynamic_jump_taken = true;
```

Comment:
The code generator can handle this, no need to record if the jump is taken at runtime. 

What is this used for?